ledger/participant-integration-api/src/main/scala/platform/apiserver/execution/LedgerTimeAwareCommandExecutor.scala

```diff
@@ -31,22 +32,24 @@ private[apiserver] final class LedgerTimeAwareCommandExecutor(
   override def execute(
       commands: Commands,
       submissionSeed: crypto.Hash,
+      ledgerConfiguration: Configuration,
```

Comment:
You may want to introduce passing the ledger configuration in a separate PR, i.e., as we don't check configurations yet by the committer anyway (this would need removal of `SubmitterInfo.ledgerConfiguration` as well or always setting it to the default).

Reply:
I could, but it would be an ignored parameter and we'd have to suppress some warnings. I was trying to avoid this.

Reply:
No problem — after merging the PRs you've broken out from this one, passing the ledger configuration should be a minimal addition.